### PR TITLE
Export submission metadata in OData output

### PR DESCRIPTION
**Pull request by clint-tseng**
See commits for more; includes the following structure (and associated schema metadata) in each root level JSON output:

```
__system: {
  submissionDate: '<iso date>',
  submitterId: '<int>',
  submitterName: '<string>'
}
```

### CODE REVIEW

File: lib/resources/odata.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ module.exports = (service, { Form, Submission, env }) => {
     Form.getByXmlFormId(params[0]) // first regexp match
       .then(getOrNotFound)
       .then((form) => auth.canOrReject('read', form)
-        .then(() => Submission.getById(form.id, params[1])
+        .then(() => Submission.getById(form.id, params[1], true)
```
Comment (lognaturel) [inline lib/resources/odata.js]: The `true` here ensures that the join on the actor is always performed so the name can be pulled but it's not possible to get the full actor object in the odata feed, right?

Comment (clint-tseng) [reply lib/resources/odata.js]: Correct. The way these records get pulled apart nothing is ever automatically passthrough.

Comment (clint-tseng) [reply lib/resources/odata.js]: And besides, actor information is really really boring. It's _user_ information that contains interesting things, like emails and password hashes. Actors are pretty much just id and displayname (and timestamp metadata).

Comment (lognaturel) [reply lib/resources/odata.js]: That makes sense. I was getting a little turned around regarding the meaning of the extended parameter vs. the extended header.

Comment (clint-tseng) [reply lib/resources/odata.js]: often the latter triggers the former!

**Comment by lognaturel** [comment]
Just realized docs should probably be updated to show this. (https://jubilantgarbanzo.docs.apiary.io/#reference/odata-endpoints/odata-form-service/data-document, etc)

**Comment by clint-tseng** [comment]
oh i suppose the examples could get updated. but the formal docs themselves don't actually even describe `__id`; the odata metadata document documents the available data. i'll update the examples, though.

**Pull request merged**